index.html

```diff
@@ -1813,8 +1813,8 @@ <h3 id="Schema-Building">Schema Building:</h3>
       is passed as the second argument.
       <br />
       Note that the method for defaulting to the current datetime varies from one database to another.
-      For example: PostreSQL requires <tt>.defaultTo(knex.raw('now()'))</tt>,
-      but SQLite3 requires <tt>.defaultTo(knex.raw("date('now')"))</tt>.
+      For example: PostreSQL requires <tt>.defaultTo(knex.fn.now())</tt>,
```

Comment:
Might be a good idea to fix the typo in "PostgreSQL" that exists in the original line. :wink: